.github/workflows/merge-check.yml

```diff
@@ -27,4 +27,4 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Run Gradle tasks
-        run: ./gradlew build
\ No newline at end of file
+        run: ./gradlew build dependencyCheckAnalyze
```

Comment:
This should not be needed though, right?

Reply:
I am thinking of the reverse - `dependencyCheckAnalyze` shouldn't be needed in `publish-snapshot` because that GH workflow is meant for snapshot publishing. This merge check is the place to add an additional check